lib/pengine/unpack.c

```diff
@@ -1931,7 +1933,7 @@ find_anonymous_clone(pe_working_set_t *data_set, const pe_node_t *node,
                 && !pcmk_is_set(child->flags, pe_rsc_block)) {
                 // Remember one inactive instance in case we don't find active
                 inactive_instance = parent->fns->find_rsc(child, rsc_id, NULL,
```

Comment:
line length, though this one is inconvenient and you probably won't consider it worth changing :)

Reply:
yeah, it's not an absolute rule :)